src/grid/Header.tsx

```diff
@@ -24,6 +26,8 @@ export interface FilterRenderer {
 }
 
 export interface HeaderProperties {
+	/** optional message bundle to override the included bundle */
+	bundle?: typeof defaultBundle;
```

Comment:
How come we need to add the `bundle` property? Doesn't this come from the I18nMixin?

Reply:
Since the `@dojo/widgets/grid/nls/Grid` bundle is controlled by the widgets team and there is currently no API for dynamically assigning locale messages to a bundle, users need a means of overriding messages, either the defaults or for locales that are not explicitly supported by `@dojo/widgets/grid/nls/Grid`. I've starting thinking about a method we could add to `@dojo/framework/i18n` that would mitigate the need for what I've done here, but in the meantime it seemed like the best solution.

Reply:
Doesn't the `i18nBundle` property allow users to override the bundle?

Reply:
Ah, I'd completely forgotten that had been added with your updates late last year. Fixed.

Reply:
I think you will still need to pass the `i18nBundle` property down through the grid widgets, so if a user does pass a custom bundle it get's used throughout the grid.

Reply:
Makes sense. Done.